config/rbac/kustomization.yaml

```diff
@@ -9,9 +9,3 @@ resources:
 - role_binding.yaml
 - leader_election_role.yaml
 - leader_election_role_binding.yaml
-# Comment the following 4 lines if you want to disable
-# the auth proxy (https://github.com/brancz/kube-rbac-proxy)
-# which protects your /metrics endpoint.
-- auth_proxy_service.yaml
-- auth_proxy_role.yaml
-- auth_proxy_role_binding.yaml
```

Comment:
You still needing RBAC permissions for the metrics 
The Controller-Runtime feature will use it at the same away that kube-rbac-proxy

See the rules scaffold by default in the Kubebuilder Sample; https://github.com/kubernetes-sigs/kubebuilder/tree/master/testdata/project-v4/config/rbac

See that you will need to have:


https://github.com/kubernetes-sigs/kubebuilder/blob/d063d5af162a772379a761fae5aaea8c91b877d4/testdata/project-v4/config/rbac/kustomization.yaml#L12-L20

Reply:
thanks, I've reintroduced that part